app/views/search/single_facility.html.erb

```diff
@@ -1,62 +1,122 @@
 <% provide(:title, "Single Facility") %>
 <div id="content" class="interior">
   <a id="content" href="#content" class="sr-only">Content starts here.</a>
-  <%= render 'layouts/breadcrumbs' %>
-</div> 
```

Comment:
*I had moved the breadcrumb into a partial. Can we use this partial instead
(so put this back in and remove lines 5-14 I believe)? (In your defense, I
made this change after you originally wrote the file, so if you just copied
it over, it would have deleted this without you knowing).*

Will this work? If the breadcrumb is in a partial, don't we lose the
ability to have the breadcrumb change for each page? For instance, if we
wanted to have the breadcrumb reflect the facility name on the facility
detail page? (i.e. Service & Facility Locator > Togus Medical Center.)

On Thu, Feb 25, 2016 at 10:15 PM, Alex Yale-Loehr <notifications@github.com>
wrote:

> In app/views/search/single_facility.html.erb
> <https://github.com/department-of-veterans-affairs/service-and-facility-locator/pull/41#discussion_r54200815>
> :
>
> > @@ -1,62 +1,122 @@
> >  <% provide(:title, "Single Facility") %>
> >  <div id="content" class="interior">
> >    <a id="content" href="#content" class="sr-only">Content starts here.</a>
> > -  <%= render 'layouts/breadcrumbs' %>
> > -</div>
>
> I had moved the breadcrumb into a partial. Can we use this partial instead
> (so put this back in and remove lines 5-14 I believe)? (In your defense, I
> made this change after you originally wrote the file, so if you just copied
> it over, it would have deleted this without you knowing).
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/department-of-veterans-affairs/service-and-facility-locator/pull/41/files#r54200815>
> .
>